src/utils/AutoDiscoveryUtils.js

```diff
@@ -22,6 +22,7 @@ import {makeType} from "./TypeUtils";
 import SdkConfig from '../SdkConfig';
 
 const LIVELINESS_DISCOVERY_ERRORS = [
+    AutoDiscovery.ERROR_INVALID,
```

Comment:
Hmm, wait why are there changes in the file...? These seem unrelated to Jitsi / widgets.

Reply:
oops :(